File: app/src/main/java/org/tvheadend/tvhclient/features/search/SuggestionProvider.java
package org.tvheadend.tvhclient.features.search;

import android.content.SearchRecentSuggestionsProvider;

public class SuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "org.tvheadend.tvhclient.features.search.SuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public SuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/search/SuggestionProvider.kt
package org.tvheadend.tvhclient.ui.features.search

import android.content.SearchRecentSuggestionsProvider

class SuggestionProvider : SearchRecentSuggestionsProvider() {

    init {
        setupSuggestions(AUTHORITY, MODE)
    }

    companion object {
        const val AUTHORITY = "org.tvheadend.tvhclient.ui.features.search.SuggestionProvider"
        const val MODE = DATABASE_MODE_QUERIES
    }
}
